ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -70,6 +70,16 @@ tests:
   commands: ./test-prow-e2e.sh
   openshift_installer_src:
     cluster_profile: gcp
+- artifact_dir: /tmp/artifacts
+  as: kubevirt-plugin
+  commands: |
+    export ARTIFACT_DIR=/tmp/artifacts
+    ./packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh
+    ./test-gui.sh kubevirt-gating
+  openshift_installer_custom_test_image:
+    cluster_profile: azure4
+    enable_nested_virt: true
+    from: console
```

Comment:
This needs to be `stable:console`, I believe.

Reply:
changed to `stable:console`. Thanks.